src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/aggregators/SlidingWindow.java

```diff
@@ -28,13 +28,11 @@
   protected final Deque<E> windowDeque;
   protected final long SLIDING_WINDOW_SIZE;
   protected double sum;
-  protected TimeUnit timeUnit;
```

Comment:
This should be fixed by an earlier PR.

Reply:
Right this is added in this one just for the build to pass